.pre-commit-config.yaml

```
@@ -42,9 +42,19 @@ repos:
         name: isort (pyi)
         types: [pyi]
 -   repo: https://github.com/psf/black
-    rev: 24.8.0
+    rev: 24.10.0
     hooks:
     -   id: black
         args:
             - "--config"
             - "./pyproject.toml"
+-   repo: https://github.com/DavidAnson/markdownlint-cli2
+    rev: v0.16.0
+    hooks:
+    -   id: markdownlint-cli2
+        name: markdownlint
+        files: "docs/*.md"
```

Yes, this doesn't work. I'm getting this warning and also no files are being found:

```
$pre-commit run --all-files
[WARNING] The 'files' field in hook 'markdownlint-cli2' is a regex, not a glob -- matching '/*' probably isn't what you want here
...
markdownlint.........................................(no files to check)Skipped
```

Try this, it works for me:
```suggestion
        files: ^docs/.*\.md$
```

---

It seems to also work with just `docs`